charts/kusion/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: kusion
-version: 0.14.1
+version: 0.14.2
 type: application
```

Comment:
The appVersion (0.14.0) appears outdated compared to chart version (0.14.2). Consider updating appVersion to match the actual application version if this chart is intended to deploy version 0.14.2 of Kusion.